onnxruntime/core/common/denormal.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if defined(_M_IX86) || defined(_M_X64) || defined(__i386__) || defined(__x86_64__)
+#define PLATFORM_X86
+#endif
+
+#if defined(PLATFORM_X86)
+#if defined(_MSC_VER)
+#include <intrin.h>
+#define DENORMAL_INTRINC
+// intrins headers at gcc 4.8 and older are not usable without compiler flags.
+// clang claims gnuc 4.2, but it has a proper intrin header.
+#elif defined(__clang__) || (defined(__GNUC__) && ((__GNUC__ >= 5) || ((__GNUC__ == 4) && (__GNUC_MINOR__ > 8))))
+#include <cpuid.h>
+#include <x86intrin.h>
+#define DENORMAL_INTRINC
+#endif
+#endif
+
+#include <mutex>
```

Comment:
nit: is <mutex> still needed? I think you had it from std::call_once in an earlier iteration.

Reply:
removed